java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java

```diff
@@ -56,6 +56,13 @@ public class ConfigDefaults {
     public static final String WEB_SESSION_SWAP_SECRET_4 = "web.session_swap_secret_4";
 
     public static final String WEB_SMTP_SERVER = "java.smtp_server";
+    public static final String WEB_SMTP_PORT = "java.smtp_port";
+    public static final String WEB_SMTP_AUTH = "java.smtp_auth";
+    public static final String WEB_SMTP_SSL = "java.smtp_ssl";
+    public static final String WEB_SMTP_STARTTLS = "java.smtp_starttls";
+    public static final String WEB_SMTP_USER = "java.smtp_user";
+    public static final String WEB_SMTP_PASS = "java.smtp_pass";
```

Comment:
You should add the variables to /java/conf/rhn_java.conf . This will go into config-defaults directory. You can add default values there and use comments to document the variables.